api/src/main/scala/hmda/api/model/EditResults.scala

```diff
@@ -20,15 +19,15 @@ trait EditResultsCollection {
   def toCsv(editType: String) = edits.map(e => e.toCsv(editType)).mkString
 }
 
-case class QualityEditsVerification(verified: Boolean)
-case class QualityEditsVerifiedResponse(verified: Boolean, status: SubmissionStatus)
+case class EditsVerification(verified: Boolean)
+case class EditsVerifiedResponse(verified: Boolean, status: SubmissionStatus)
 
-case class MacroResult(edit: String, justifications: Set[MacroEditJustification])
-case class MacroResults(edits: Seq[MacroResult]) {
+case class MacroResult(edit: String)
+case class MacroResults(verified: Boolean, edits: Seq[MacroResult]) {
   def toCsv = edits.map(e => "macro, " + e.edit + "\n").mkString
 }
 case object MacroResults {
-  def empty: MacroResults = MacroResults(Nil)
+  def empty: MacroResults = MacroResults(true, Nil)
```

Comment:
Good call! That object is indeed unused, so I deleted it in 599078734ec9170adfc5cb1d3cbc5c3361fa9aa2.